dungeon/src/task/quizquestion/QuizQuestion.java

```diff
@@ -0,0 +1,153 @@
+package task.quizquestion;
+
+import task.Task;
+import task.TaskContent;
+
+/**
+ * Represents a single quiz question, including the question itself, possible answer choices, and
+ * the type of question.
+ *
+ * <p>A {@link QuizQuestion} can be a Single-Choice, a Multiple-Choice, or a Free-text question. The
+ * type is stored as a {@link QuizType}. If the question is asked via the UI, the {@link QuizUI}
+ * will configure the UI for the question based on that type. The type can be accessed via {@link
+ * #type()}.
+ *
+ * <p>Add a {@link QuizContent} answer by using the {@link #addAnswer(QuizContentType, String)}
+ * method. Use the {@link #contentStream()} method to get the answers as a stream.
+ *
+ * <p>The question will be stored as {@link QuizContent} and can be accessed via {@link
+ * #question()}.
+ */
+public class QuizQuestion extends Task {
+
+    private final QuizType type;
+    private final QuizContent question;
+
+    /**
+     * Create a new {@link QuizQuestion} with the given configuration.
+     *
+     * <p>This will create a new {@link QuizContent} instance as the question reference.
+     *
+     * <p>The {@link QuizQuestion} will not have any answers, use {@link #addAnswer(QuizContentType,
+     * String)} to add possible answers to the question.
+     *
+     * @param type Type of the question (e.g., single-choice)
+     * @param questionContentType What does the question contain? Just text, just a path to an
+     *     image, or both text and a path to an image?
+     * @param questionText The question itself (can contain a path to images).
+     */
+    public QuizQuestion(
+            final QuizType type, QuizContentType questionContentType, String questionText) {
+        super();
+        this.type = type;
+        taskText(questionText);
+        question = new QuizContent(this, questionContentType, questionText);
```

Comment:
an der stelle habe ich überlegt, ob es nicht besser wäre in `Task#taskText` besser als `TaskContent` Objekt zu speichern. 
In diesen konkreten Fall brauche ich, um der Fragestellung einen Typen anzuheften, damit ich weiß ob im der Fragestellung ein Pfad zu einem Bild gespeichert ist und ich diesen im UI extrahieren muss. 

Ich kann aber nicht einschätzen, ob das für die anderen Aufgabentypen auch sinnvoll ist, daher hab ich es erstmal gelassen, wollte es hier aber einmal erwähnen.

Reply:
Das Gefühl hatte mich neulich auch schon mal beschlichen - brauchen wir wirklich zu jedem TaskType auch das exakt passende TaskContentType? Oder gibt es ein paar ContentType(n), mit denen man dann was hübsches für die einzelnen Tasks zusammenbauen kann? Würde auch sofort die ganzen Querverstrebungen aka Abhängigkeiten reduzieren ...

Reply:
Ich würde an der Stelle mal ein neues Issue aufmachen, um den Gedanken festzuhalten. 
Dann lassen wir es hier so und passen es ggf an.

Reply:
#862